server/services/oneThousandCalculationsService.ts

```diff
@@ -265,6 +303,28 @@ export default class OneThousandCalculationsService {
     return ''
   }
 
+  isHdced14DayRule(breakdown: CalculationBreakdown, calc: BookingCalculation): 'Y' | 'N' | '' {
+    if (calc?.dates.HDCED) {
```

Comment:
could use ternary operator to replace 389 to 32
`return breakdown?.breakdownByReleaseDateType?.HDCED?.rules?.includes('HDCED_MINIMUM_14D') ? 'Y' : 'N'`